deploy/staging/chainlog-logger.yaml

```diff
@@ -0,0 +1,64 @@
+replicaCount: 1
+service:
+  name: chainlog-logger
+  port: 8080
+  type: ClusterIP
+  containerPort: 8080
+  tls:
+    enabled: false
+image:
+  repository: 068992353948.dkr.ecr.eu-central-1.amazonaws.com/chainlog-logger-staging
+  pullPolicy: IfNotPresent
+  tag: staging
+serviceAccount:
```

Comment:
We should be using these in the future, I imagine you haven't set it up yet

Reply:
@eskp no, we have this functionality, please have a look [here](https://github.com/MakerOps/helm-charts/blob/main/charts/common/templates/serviceAccount.yaml)
It is just that Chainlog UI and Logger don't need any access to AWS resources